apps/rd/rd-judicial-data-load/demo-01.yaml

```diff
@@ -8,3 +8,5 @@ spec:
       schedule: "15 12 * * *"
       environment:
         DELETE_ORG: true
+    java:
```

Comment:
```suggestion
```

Reply:
thanks @timja , removed the java and moved under job